### ISSUE TITLE
Feature request: Search show/episode details

### ISSUE THREAD

**Issue by source124** [ltguillaume/droidshows#42]
Hi,

Thanks for the great app :)

It would be great if one could find a certain episode by searching for a keyword in the synopsis or a guest actor for example.

Thank you.

**Comment by Ibuprophen** [comment]
That is a good idea as a feature request/suggestion but, that would require ALLOT of Coding to add a Search/Query feature within already added Shows/Seasons/Episodes...

**Comment by source124** [comment]
@ibuprophen1 

> That is a good idea as a feature request/suggestion but, that would require ALLOT of Coding to add a Search/Query feature within already added Shows/Seasons/Episodes...

Could you explain why it would be so? It does not have to be semantic.

**Comment by Ibuprophen** [comment]
I recall asking ltGuillaume regarding a similar/same feature a while ago (i know it was 30+ days ago but, i can't recall exactly when) and it wasn't discounted as a possibility but, ltGuillaume did express (in so many words) the same statement i had provided.

I apologize for not elaborating on this within the last comment. I never intended it to be so blunt.

Thank you!

**Comment by source124** [comment]
Thanks for your reply :)

I did a search before I opened the issue and I couldn't find similar feature requests. Would you provide a link, please?

Thanks

**Comment by ltguillaume** [comment]
It wasn't the same request. I'll look into this, because I really like this idea. It will indeed cost me time, so first I'll try to get TheTVDB's APIv2 working.

**Comment by source124** [comment]
Thank you :)

**Comment by ibuprophen1** [comment]
This would be a GREAT feature too!

**Comment by Ibuprophen** [comment]
This would indeed be a great feature! 

The ability to search within the already added Shows for certain keywords located in the Title, Episode Name and even the Episode Description (as long as there's information populated in there) would be an additional bonus. 

Just a suggestion... LMAO!

**Comment by ltguillaume** [comment]
Yeah I'll redo the database structure when I finish the APIv2 implementation: I'll remove some info that's never presented and I'll stop putting the actors in a separate table. This way I'll save quite a bit of queries (also when updating, so it'll be faster) and it'll be much easier to implement this feature properly.

**Comment by Ibuprophen** [comment]
I wasn't sure if you were aware of this @ltGuillaume but, TheTVDB folks just posted the following on their forum that, I believe, may be of interest to you My Friend! :-) 

Announcement: GitLab All the (APIv2) Things!
https://forums.thetvdb.com/viewtopic.php?f=17&t=47792


~Ibuprophen

**Comment by ltguillaume** [comment]
Thx!

**Comment by Flexmaen** [comment]
Also checkt for this feature request and saw it is already here!

However, thanks for that great app!
